tests/pytorch/r2.1/common.libsonnet

```diff
@@ -101,7 +101,7 @@ local volumes = import 'templates/volumes.libsonnet';
         #   https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch-2.1-cp310-cp310-linux_x86_64.whl \
         #   'torch_xla[tpuvm] @ https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.1-cp310-cp310-linux_x86_64.whl'
         pip install --user --pre --no-deps torch torchvision --extra-index-url https://download.pytorch.org/whl/nightly/cpu
```

Comment:
Can you change this to the upstream rc2 wheel too? https://dev-discuss.pytorch.org/t/pytorch-2-1-rc1-is-produced-for-pytorch-vision-and-audio/1479/2

I think this is the one we want from their index: https://download.pytorch.org/whl/test/cpu/torch-2.1.0%2Bcpu-cp310-cp310-linux_x86_64.whl

Reply:
Updated